### PR TITLE
Update code samples

### PR DESCRIPTION
**Pull request by luise**
They'd gotten out of date with all the recent API changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -266,13 +266,13 @@ <h2 class="text-center">Quilt in Action</h2>
 							<div class="col-xs-10 col-sm-11 code">
 								<ul>
 									<li>
-										<span>var vm = </span><span class="color-red">new </span><span>Machine({</span>
+										<span>let vm = </span><span class="color-red">new </span><span>Machine({</span>
```
Comment (kayousterhout) [inline index.html]: why not const here?

File: index.html [hunk 2]
```diff
@@ -231,7 +231,7 @@ <h2 class="text-center">Quilt in Action</h2>
 										<span class="color-red">const </span><span>mongo = </span><span class="color-red">new </span><span>Mongo(</span><span class="color-blue">2</span><span>);</span><span class="color-comment"> // Two MongoDB Replicas.</span>
 									</li>
 									<li>
-										<span class="color-red">const </span><span>haproxy = </span><span class="color-red">new </span><span>HaProxy(</span><span class="color-blue">2</span><span>, node);</span><span class="color-comment"> // Two load balancers over Node.js containers</span>
+										<span class="color-red">const </span><span>proxy = haproxy.<span class="color-red">singleServiceLoadBalancer</span>(</span><span class="color-blue">2</span><span>, node);</span><span class="color-comment"> // Two load balancers over Node.js containers</span>
```
Comment (kayousterhout) [inline index.html]: should this be simpleLoadBalancer?

Comment (luise) [reply index.html]: Oh, yes, we should update the HAproxy README -- I just copied the constructor from there. I'll update it!

File: index.html (hunk 1 above)
Comment (luise) [reply index.html]: Ah, yes, in my mind when I wrote this, `const` meant that the value wouldn't be edited (which we sometimes do with VMs), but I just mixed that up -- it means that the variable won't be reassigned. Will fix!

**Comment by luise** [comment]
Ugh, our example doesn't quite make sense anymore with the new API. Basically the LoadBalancer is redundant now that HAProxy takes a list of containers instead of a Service.
It would make sense if we just remove the `new LoadBalancer`, but I don't know how much we care about showing that feature? I'd say we should just remove if from the example for now, and then we can find something more suitable when we make a new website.

**Comment by kayousterhout** [comment]
Sounds good re:just removing new LoadBalancer!

**Comment by luise** [comment]
@kayousterhout , updated.

**Pull request merged**